### PR TITLE
 Added 'Intro to Robotics' syllabus

### PR DESCRIPTION
**Pull request by La-Nique**

### CODE REVIEW

**Comment by La-Nique** [comment]
^___^ beep boop incoming!

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@ _site/
 .jekyll-cache/
 .jekyll-metadata
 Gemfile.lock
+assets/courses/.DS_Store
```
Comment (joshnatis) [inline .gitignore:6]: 🤣 lmfaooo, yeah I think this will potentially prevent some annoyance for some people, let's go with it :D

**Review by joshnatis — APPROVED** [review]
Looks good!! Give your friend a big thank you pls and ask them again if they want to be in the README 😎. It's not just for fame and glory, it makes this project feel more grassroots + helps ppl discover fellow Hunter students.

File: .gitignore (hunk 1 above)
Comment (La-Nique) [reply .gitignore:6]: :D !!

**Pull request merged**